view/adminhtml/templates/config/applepay_button.phtml

```diff
@@ -1,3 +1,3 @@
 <?php /** @var \Adyen\Payment\Model\Config\Adminhtml\ApplepayCertificateButton $block */ ?>
 
-<input type="button" onclick="location.href='<?= $block->getActionUrl() ?>';" value="Download certificate" />
+<input type="button" onclick="location.href='<?= $block->escapeJs($block->getActionUrl()) ?>';" value="Download certificate" />
```

Comment:
```suggestion
<input type="button" onclick="location.href='<?= $block-> escapeUrl($block->getActionUrl()) ?>';" value="Download certificate" />
```

Reply:
`onclick` attribute is a part of the JS code although it's a URL. According to the documentation, `escapeUrl()` can be safely used on `href` and `src` html attributes. But, any code which is part of a JS script needs to be escaped with `escapeJs()` function. Below example was taken from the documentation.

```
<script>
  let phrase = "Hi, my name is <?= $escaper->escapeJs($myName) ?>";
  //Do not use HTMl context methods like escapeUrl
  let redirectUrl = "<?= $escaper->escapeJs($myUrl) ?>";
  location.href = redirectUrl;
</script>
```

[Reference](https://developer.adobe.com/commerce/php/development/security/cross-site-scripting/)